webpack.config.js

```diff
@@ -157,7 +157,11 @@ function appConfig(options, argv) {
                         },
                         {
                             test: /app\/polyfill\.ts$/,
-                            include: join(__dirname, 'packages', 'core', 'src'),
+                            include: [
+                                    join(__dirname, 'packages', 'core', 'src'),
+                                    join(__dirname, 'packages', 'locale', 'src'),
+                                    join(__dirname, 'packages', 'test-utils', 'src'),
```

Comment:
If `test-utils` is not included, the following error will show up during the build process.
```node
ERROR in ./packages/test-utils/src/payment-methods.mock.ts 3:34
Module parse failed: Unexpected token (3:34)
File was processed with these loaders:
 * ./node_modules/source-map-loader/index.js
You may need an additional loader to handle the result of these loaders.
| import { PaymentMethod } from '@bigcommerce/checkout-sdk';
| 
> export function getPaymentMethod(): PaymentMethod {
|     return {
|         id: 'authorizenet',
 @ ./packages/test-utils/src/index.ts 6:0-10:32 6:0-10:32 6:0-10:32 6:0-10:32
 @ ./packages/locale/src/localeContext.mock.tsx 3:0-66 7:36-50
 @ ./packages/locale/src/index.ts 14:0-56 14:0-56
 @ ./packages/core/src/app/locale/index.ts 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299 1:0-299
 @ ./packages/core/src/app/loader.ts 12:0-68 43:23-39 48:22-44
```